common/changes/@microsoft/gulp-core-build/sajosep-upgradeJest_2020-04-29-07-57.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Upgrade Jest to 25",
```

Comment:
This is the only changefile that should have a comment and a bump type. The rest shouldn't have a comment and the bump type should be `"none"`